features/rageshake/src/main/kotlin/io/element/android/features/rageshake/logs/VectorFileLogger.kt

```diff
@@ -101,14 +104,14 @@ class VectorFileLogger(
     fun reset() {
         // Delete all files
         getLogFiles().map {
-            tryOrNull { it.delete() }
+            it.safeDelete()
         }
     }
 
     @OptIn(DelicateCoroutinesApi::class)
     override fun log(priority: Int, tag: String?, message: String, t: Throwable?) {
         fileHandler ?: return
-        GlobalScope.launch(Dispatchers.IO) {
+        GlobalScope.launch(dispatcher) {
```

Comment:
If we inject dispatcher we can also inject the app CoroutineScope and remove GlobalScope